.travis.yml

```diff
@@ -17,11 +17,12 @@ language: python
 python:
   - '2.7'
 env:
-  - PYARROW_VERSION=0.12.1 TF_VERSION=1.12 DEPLOYABLE=0 PY=2.7 PYSPARK_VERSION=2.4.4
-  - ARROW_PRE_0_15_IPC_FORMAT=1 PYARROW_VERSION=0.15.1 TF_VERSION=1.12 DEPLOYABLE=0 PY=2.7 PYSPARK_VERSION=2.4.4
-  - PYARROW_VERSION=0.12.1 TF_VERSION=1.13.2 DEPLOYABLE=0 PY=3.7 PYSPARK_VERSION=2.4.4
```

Comment:
@selitvin compat.v1 was introduced in 1.14. So this PR would deprecate the support for TF < 1.14. Is it fine?

@WeichenXu123 Does CI work with TF 1.14?

Reply:
Should update setup.py if we only support tf >= 1.14.